Makefile

```
@@ -17,6 +17,14 @@ static-server:
 # Prod server - persistent background process without process manager or Docker
 # Uses sudo to bind to port 80 and/or 443
 
+# Docker
+
+docker-build:
+	docker image build --build-arg DOPPLER_TOKEN=$(DOPPLER_TOKEN) -t yodaspeak:latest .
```

We can publish these images under the existing `dopplerhq` org (like with our [CLI](https://hub.docker.com/r/dopplerhq/cli)).

```suggestion
	docker image build --build-arg DOPPLER_TOKEN=$(DOPPLER_TOKEN) -t dopplerhq/yodaspeak:latest .
```